corehq/apps/linked_domain/views.py

```diff
@@ -278,7 +278,10 @@ def page_context(self):
             self.domain, master_apps, master_fixtures, master_reports, master_keywords, is_superuser=is_superuser
         )
 
-        available_domains_to_link = get_available_domains_to_link(self.request.domain, self.request.couch_user)
+        current_subscription = Subscription.get_active_subscription_by_domain(self.request.domain)
```

Comment:
FYI there's a `BillingAccount.get_account_by_domain` does this same logic.

Reply:
good point